antenna/throttler.py

```diff
@@ -138,7 +154,9 @@ def __init__(self, rule_name, key, condition, percentage):
         """
         self.rule_name = rule_name
         self.key = key
-        self.condition = self._to_handler(condition)
+        if not callable(condition):
+            raise ValueError('condition %r is not callable' % condition)
```

Comment:
This will never happen in production. However, it makes it muuuuuch easier to debug code and it makes it more likely all our issues will pop up when running the tests.